html/elements/frame.json

```diff
@@ -0,0 +1,57 @@
+{
+  "html": {
+    "elements": {
+      "frame": {
```

Comment:
Do we want to have the attributes here? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/frame#Attributes

Reply:
Yes, fixed.